documentation/puppetdb_connection.markdown

```diff
@@ -101,7 +101,7 @@ The default value is false.
 
 When using multiple `server_urls`, this flag can be set to `true` to cause commands to be sent to all configured PuppetDB instances.
 
-The default value is false.
+The default value is `false` when not specified, but Puppet Enterprise 2016.5 and later explicitly configures `command_broadcast` to `true`.
```

Comment:
Would it be clearer to say something like "Defaults to 'false' in FOSS and 'true' in Puppet Enterprise > 2016.5". To me the current wording implies that what PE is doing is something other than defaulting, which is a little confusing.